.bazelrc

```diff
@@ -1,5 +1,8 @@
+# Sadly, some Bazel rules we depend on have no support for bzlmod yet
```

Comment:
Can we document them so that it's easier to check for support later?

Reply:
I added a link to the upstream rules_nixpkgs issue. Also mentioned that there may be more we are not seeing yet.